build/BUILD.sqlite3

```diff
@@ -8,5 +8,7 @@ cc_library(
     defines = [
         "SQLITE_MAX_ALLOCATION_SIZE=16777216",  # 16MB
         "SQLITE_PRINTF_PRECISION_LIMIT=100000",
+        "SQLITE_ENABLE_UPDATE_DELETE_LIMIT=1", # https://sqlite.org/draft/compile.html#enable_update_delete_limit
```

Comment:
According to the docs, this won't work:

> If this option is defined, then it must also be defined when using the [Lemon parser generator](https://sqlite.org/draft/lemon.html) tool to generate a parse.c file. Because of this, this option may only be used when the library is built from source, not from the [amalgamation](https://sqlite.org/draft/amalgamation.html) or from the collection of pre-packaged C files provided for non-Unix like platforms on the website.

We build from the amalgamation.